ChangeLog

```diff
@@ -1,3 +1,8 @@
+2017-08-04    <adam.johansen@gmail.com>
```

Comment:
The format is date, two spaces, name, two spaces, email.

That said, it is also uncommon to write these for someone else.